### PR TITLE
allow selecting multiple columns with SELECT DISTINCT

### PR DESCRIPTION
**Pull request by icewind1991**
Signed-off-by: Robin Appelman <robin@icewind.nl>

### CODE REVIEW

File: tests/lib/DB/QueryBuilder/QueryBuilderTest.php [hunk 1]
```diff
@@ -314,6 +314,34 @@ public function testSelectDistinct() {
 		$this->deleteTestingRows('testFirstResult2');
 	}
 
+	public function testSelectDistinctMultiple() {
+		$this->deleteTestingRows('testFirstResult1');
+		$this->deleteTestingRows('testFirstResult2');
+		$this->createTestingRows('testFirstResult1');
+		$this->createTestingRows('testFirstResult2');
+
+		$this->queryBuilder->selectDistinct(['appid', 'value']);
```
Comment (nickvergessen) [inline tests/lib/DB/QueryBuilder/QueryBuilderTest.php:323]: Can't select distinct a clob on oracle 💥

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by MorrisJobke — APPROVED** [review]
Code makes sense 👍

**Comment by nickvergessen** [comment]
Let me run this on oracle tomorrow

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**